tests/modules/moduleparameter.c

```diff
@@ -0,0 +1,25 @@
+#include "valkeymodule.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+int GET_HELLO(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
+  ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);
+  return ValkeyModule_ReplyWithSimpleString(ctx, "Module runtime args test");
```

Comment:
```suggestion
    ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);
    return ValkeyModule_ReplyWithSimpleString(ctx, "Module runtime args test");
```
I guess we should probably apply the formatter on these files as well.